Sharpmake.Generators/Apple/XCodeProj.cs

```diff
@@ -336,7 +336,10 @@ string projectFile
                 Options.Option(Options.XCode.Scheme.MetalAPIValidation.Disable, () => options["MetalAPIValidation"] = "1")
             );
 
-            string targetName = $"&quot;{activeConfiguration.Target.Name}&quot;";
+            options["CustomDirectory"] = Options.PathOption.Get<Options.XCode.Scheme.CustomWorkingDirectory>(activeConfiguration);
+            var useCustomDirectory = options["CustomDirectory"] != RemoveLineTag ? "YES" : "NO";
+
+            string targetName = $"{activeConfiguration.Target.Name}";
```

Comment:
Please extract the target name change in a separate PR

Reply:
Sorry for the delay. Fixed.